Translate the overview of Flutter for ios-dev

wanbok opened the pull request:
아래 내용들이 다 번역되기 전에도 잦은 피드백을 주시면 좋겠습니다.

# Flutter for iOS developers
- [x] Overview
## Views
- [x] What is the equivalent of a `UIView` in Flutter?
- [x] How do I update `Widget`s?
- [x] How do I lay out my widgets? Where is my Storyboard?
- [x] How do I add or remove a component from my layout?

-----
# TODOs
- [ ] How do I animate a Widget?
- [ ] How do I draw to the screen?
- [ ] Where is the widget's opacity?
- [ ] How do I build custom widgets?
## Navigation
- [ ] How do I navigate between pages?
- [ ] How do I navigate to another app?
- [ ] How do I pop back to the iOS native viewcontroller?
- [ ] How do I write asynchronous code?
- [ ] How do you move work to a background thread?
- [ ] How do I make network requests?
- [ ] How do I show the progress of a long-running task?
## Project structure, localization, dependencies and assets
- [ ] How do I include image assets for Flutter? What about multiple resolutions?
- [ ] Where do I store strings? How do I handle localization?
- [ ] What is the equivalent of CocoaPods? How do I add dependencies?
## Layouts
- [ ] What is the equivalent to `ViewController`s in Flutter?
- [ ] How do I listen to iOS lifecycle events?
- [ ] What is the equivalent of a `UITableView` or `UICollectionView` in Flutter?
- [ ] How do I know which list item is clicked?
- [ ] How do I dynamically update a `ListView`s?
- [ ] What is the equivalent of a `ScrollView` in Flutter?
## Gesture detection and touch event handling
- [ ] How do I add a click listener to a widget in Flutter?
- [ ] How do I handle other gestures on widgets?
## Theming and text
- [ ] How do I theme an app?
- [ ] How do I set custom fonts on my `Text` widgets?
- [ ] How do I style my `Text` widgets?
## Form input
- [ ] How do forms work in Flutter? How do I retrieve user input?
- [ ] What is the equivalent of a placeholder in a text field?
- [ ] How do I show validation errors?
## Interacting with hardware, third party services and the platform
- [ ] How do I interact with the platform, and with platform native code?
- [ ] How do I access the GPS sensor? / the camera?
- [ ] How do I log in with Facebook?
- [ ] How do I use Firebase features?
- [ ] How do I build my own custom native integrations?
## Databases and local storage
- [ ] How do I access `UserDefaults` in Flutter?
- [ ] What is the equivalent to CoreData in Flutter?
## Notifications
- [ ] How do I set up push notifications?

Commit:
Translate the overview of Flutter for ios-dev

Commit:
Translate What is the equivalent fo a 'UIView' Flutter in Views

Commit:
Translate How do I update 'Widget's

Commit:
Tranlsate How do I lay out my widgets? Where is my Storyboard?

techhtml commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
-knowledge to build mobile apps with Flutter. If you understand the
-fundamentals of the iOS framework then you can use this document as a
-way to get started learning Flutter development.
+Flutter로 모바일 앱을 만들 때 자신의 지식을 활용하려는 iOS 개발자들을 위한 문서입니다.
```
자신의 iOS 지식을 활용하려는

techhtml commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
-fundamentals of the iOS framework then you can use this document as a
-way to get started learning Flutter development.
+Flutter로 모바일 앱을 만들 때 자신의 지식을 활용하려는 iOS 개발자들을 위한 문서입니다.
+iOS 프레임워크의 원리를 이해한다면, 이 문서를 통해 Flutter 개발을 배우기 시작할 수 있어요.
```
Fundamentals 니까 원리보다는 기초라고 하는 게 좋을 거 같네요.
어조는 전체적으로 맞추면 좋을 거 같습니다.

`개발을 배우기 시작할 수 있습니다.` 정도로

techhtml commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
-the [Flutter Youtube channel](https://www.youtube.com/flutterdev) about
-the Cupertino package.
+본격적으로 뛰어들기 전에 [Flutter Youtube 채널](https://www.youtube.com/flutterdev)에서
+15분 분량의 Cupertino 패키지 비디오를 시청해보세요.
```
Cupertino 패키지에 대한 비디오

techhtml commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
-but it has a plugin system to communicate with iOS (and Android) for non-UI
-tasks. If you're an expert in iOS development, you don't have to relearn
-everything to use Flutter.
+iOS 지식과 기술셋은 Flutter 개발에 큰 도움이 되는데, Flutter는 모바일 운영체제의
```
Flutter는 다양한 기능과 구성을 위해 모바일 운영 체재를 사용하므로,
iOS 지식과 스킬 셋은 Flutter 앱을 만들 때 많은 도움이 됩니다.

techhtml commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
 
-Flutter also already makes a number of adaptations in the framework for you 
-when running on iOS. For a list, see [Platform adaptations](/docs/resources/platform-adaptations).
+Flutter는 이미 iOS에 동작시키기 위한 수 많은 어댑터들 만들었습니다. 목록은
```
어댑터 (Adaptation) 로 동시표기하면 좋을 거 같습니다.

어댑터들 -> 어댑터들을

techhtml commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
-traditional imperative style?
-For a comparison, see [Introduction to declarative
-UI](/docs/get-started/flutter-for/declarative).
+반응형 혹은 _선언형_ 프로그래밍과 기존 명령형 프로그래밍은 어떤 차이가 있을까요?
```
여기서 말하는 react-style은 반응형이라기 보다는 React 스타일을 의미하는 거 같습니다.

wanbok replied:
아 이제와서 보니 그러네요 ㅋㅋ

techhtml commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
-flexible design system [optimized for all
-platforms]({{site.material}}/design/platform-guidance/cross-platform-adaptation.html#cross-platform-guidelines),
-including iOS.
+iOS에서, UI를 만드는 것의 대부분은 `UIView` 클래스의 인스턴스인 뷰 개체를 사용합니다.
```
Object는 주로 객체라고 번역하는 편인데,
개체라고 번역하신 이유가 있을까요?

wanbok replied:
https://social.msdn.microsoft.com/Forums/ko-KR/c095ed0e-0f92-4c1b-8251-5950b4a1ca14/ctip-4406052404204913963650752-4406152404234583963651032?forum=visualcsharpko
위 내용을 접한 뒤로는 이렇게 번역하는 게 맞다고 생각하고 있습니다. 실제로 MS의 제품들은 object를 개체로 번역하고 있습니다.
https://www.microsoft.com/ko-kr/language/Search?&searchTerm=Object&langID=457&Source=true&productid=0

techhtml commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
-On iOS, you can use the [Cupertino widgets](/docs/development/ui/widgets/cupertino)
-to produce an interface that looks like
-[Apple's iOS design language](https://developer.apple.com/design/resources).
+Flutter 에서는, `UIView`와 대략적으로 같은 것이 `위젯(Widget)`입니다. 위젯이 정확하게 iOS 뷰와
```
`대략적으로 같은 것` 보다는 `유사한 것` 이라고 하는 게 좋을 거 같습니다.

techhtml commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
+뷰가 아니고, 직접적으로 그리는 것도 없습니다. 다만 위젯은 UI의 설명이자 문법이고, 내부적으로 실제 뷰에
+"부풀어 오르게" 됩니다.
+
+Flutter는 [머티어리얼 구성요소]({{site.material}}/develop/flutter/)라이브러리를
```
Material 컴포넌트

techhtml commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
+"부풀어 오르게" 됩니다.
+
+Flutter는 [머티어리얼 구성요소]({{site.material}}/develop/flutter/)라이브러리를
+포함하고 있습니다. 이 위젯들은 [머티어리얼 디자인 가이드라인]({{site.material}}/design/)
```
Material 디자인 가이드라인

techhtml commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
+되어있습니다.
+
+Flutter는 다른 디자인을 구현하기에도 충분히 유연하고 표현력있습니다. iOS 에서,
+[쿠퍼티노 위젯](/docs/development/ui/widgets/cupertino)을 사용하여
```
cupertino 위젯

techhtml commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
+
+Flutter는 다른 디자인을 구현하기에도 충분히 유연하고 표현력있습니다. iOS 에서,
+[쿠퍼티노 위젯](/docs/development/ui/widgets/cupertino)을 사용하여
+[애플의 iOS 디자인 가이드](https://developer.apple.com/design/resources)처럼 보이도록
```
에서 소개하는 것처럼

techhtml reviewed (commented):
리뷰 남겼습니다!

techhtml commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -173,14 +154,12 @@ class _SampleAppPageState extends State<SampleAppPage> {
 {% endprettify %}
 
-### How do I lay out my widgets? Where is my Storyboard?
+### 어떻게 위젯의 레이아웃을 구성할까요? 스토리보드는 어
```
번역이 중간에 사라졌네요.

techhtml commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -173,14 +154,12 @@ class _SampleAppPageState extends State<SampleAppPage> {
-constraints, or you might set your constraints programmatically in your view
-controllers. In Flutter, declare your layout in code by composing
-a widget tree.
+iOS에서, 뷰를 구성하고 뷰의 제약 설정하는데 아마도 스토리보드를 사용할 겁니다. 아니면, 그냥 코드로
```
뷰를 구성하고 제약사항을 설정하는 데

techhtml commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -173,14 +154,12 @@ class _SampleAppPageState extends State<SampleAppPage> {
+작성했을 수도 있죠. Flutter에서는 코드상에 위젯 트리를 구성하여 레이아웃을 정의합니다.
 
-The following example shows how to display a simple widget with padding:
+아래는 패딩을 포함하는 간단한 위젯의 예시입니다:
```
패딩은 그냥 padding 이라고 해도 될 거 같아요.

techhtml reviewed (commented):
리뷰 남겼습니다!

Commit:
Translate How do I add or remove a component from my layout?

techhtml commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -186,16 +186,14 @@ Widget build(BuildContext context) {
-control that child's creation with a boolean flag.
+iOS 에서는 부모 뷰에서 `addSubview()`나 `removeFromSuperview()`를 호출하여 자식 뷰를 동적으로
+추가하고 제거할 수 있습니다. Flutter 에서 위젯은 불변이기 때문에 `addSubview()`와 동일한 방법은
+없습니다. 대신에, 함수를 호출함으로써 자식 위젯을 리턴하는 부모 위젯이, boolean 값에 따라 자식 위젯들을
```
대신 위젯을 반환하는 부모에서 함수를 전달할 수 있으며,
boolean 값에 따라 자식의 생성을 제어할 수 있습니다.

techhtml commented:
@wanbok 님. 이거 덩어리가 너무 커서요.
PR 날리신 거까지만 먼저 반영하고 머지해주시고,
나머지는 추가로 PR을 날려주시는 방향이 좋을 거 같습니다.

Commit:
Apply @techhtml's reviews

wanbok commented:
@techhtml 좋습니다. 다른 분들도 리뷰/머지 부탁드려용. :)

bro-jo commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
-fundamentals of the iOS framework then you can use this document as a
-way to get started learning Flutter development.
+Flutter로 모바일 앱을 만들 때 자신의 iOS 지식을 활용하려는 개발자들을 위한 문서입니다.
+iOS 프레임워크의 기초를 이해한다면, 이 문서를 통해 Flutter 개발을 배우기 시작할 수 있습니다.
```
아래와 같이 하면 조금 더 자연스러울 것 같습니다.
지금도 좋습니다.

```suggestion
iOS 프레임워크의 기초를 이해하고 있다면, 이 문서를 통해 Flutter 개발 학습을 시작할 수 있습니다.
```
```suggestion
iOS 프레임워크의 기초를 알고 있다면, 이 문서를 통해 Flutter 개발을 배우기 시작할 수 있습니다.
```

bro-jo commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
+Flutter는 다양한 기능과 구성을 위해 모바일 운영 체재를 사용하므로, iOS 지식과 스킬 셋은
+Flutter 앱을 만들 때 많은 도움이 됩니다. Flutter는 모바일을 위한 UI 새로운 제작 방법이지만,
+iOS(그리고 안드로이드)의 UI가 아닌 작업들과 통신하기 위한 플러그인 시스템도 가지고 있습니다.
+당신이 iOS 개발 전문가라면, Flutter를 사용하는 모든 방법을 다시 배울 필요는 없습니다.
```
```suggestion
당신이 iOS 개발 전문가라면, Flutter를 사용하기 위해 모든 걸 다시 배울 필요는 없습니다.
```

bro-jo commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
-traditional imperative style?
-For a comparison, see [Introduction to declarative
-UI](/docs/get-started/flutter-for/declarative).
+React 스타일 혹은 _선언형_ 프로그래밍과 기존 명령형 프로그래밍은 어떤 차이가 있을까요?
```
이전 번역에서는 `react-style`은 그냥 원문 그대로 뒀었습니다.
이 부분 논의가 필요할 것 같아요

wanbok replied:
이미 앞선 번역이 있다면, 따라도 상관 없을 것 같습니다. cc @techhtml

bro-jo commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
-On iOS, you can use the [Cupertino widgets](/docs/development/ui/widgets/cupertino)
-to produce an interface that looks like
-[Apple's iOS design language](https://developer.apple.com/design/resources).
+Flutter 에서는, `UIView`와 유사한 것이 `위젯(Widget)`입니다. 위젯이 정확하게 iOS 뷰와
```
```suggestion
Flutter 에서는 `위젯(Widget)`이 `UIView`와 유사합니다. 위젯이 정확하게 iOS 뷰와
```

bro-jo commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
-to produce an interface that looks like
-[Apple's iOS design language](https://developer.apple.com/design/resources).
+Flutter 에서는, `UIView`와 유사한 것이 `위젯(Widget)`입니다. 위젯이 정확하게 iOS 뷰와
+같지는 않지만, Flutter가 어떻게 동작하는지 익히는 동안 위젯을 `UI를 선언하고 구성하는 방법`으로
```
```suggestion
같지는 않지만, Flutter가 어떻게 동작하는지 익히는 동안 위젯을 "UI를 선언하고 구성하는 방법"으로
```

bro-jo commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
+불변이며 변화가 생기기 전까지만 존재합니다. 위젯 혹은 위젯의 상태가 변하면, Flutter 프레임워크가
+새로운 위젯 인스턴트 트리를 생성합니다. 비교해보자면, iOS 뷰는 상태가 변해도 재생성 되지 않습니다.
+더 자세히 말하자면, iOS 뷰는 변할 수 있는 개체(mutable entity)이고, 한 번 그려진 뒤,
+`setNeedsDisplay()`를 사용하여 무효화 하는 방식으로 다시 그릴 수 있습니다.
```
```suggestion
`setNeedsDisplay()`를 사용하여 무효화하지 않는 이상 다시 그려지지 않습니다.
```

bro-jo commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
+
+더 나아가 `UIView`와 다르게, Flutter의 위젯은 불변성 덕분에 가볍습니다. 왜냐하면 위젯 스스로는
+뷰가 아니고, 직접적으로 그리는 것도 없습니다. 다만 위젯은 UI의 설명이자 문법이고, 내부적으로 실제 뷰에
+"부풀어 오르게" 됩니다.
```
안드로이드 번역할 때 문장과 거의 유사한 것 같네용 저는 이렇게 했었습니다.

```
Flutter의 위젯은 불변하기 때문에 가볍습니다.
위젯이 그 자체로 뷰가 아니기에 어떤 것도 직접 그리지 않고,
대신 UI에 대한 설명이며 내부적으로 이미 "inflate"된 실제 뷰 객체 UI의 의미론(semantics)이기 때문입니다.
```

지금 보니 첫 줄부터 이상하네요.. `불변이니`가 맞을 것 같은데 말이죠.
마지막 줄이 참 어려웠던 기억이 나요..ㅎㅎ

bro-jo commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
+것처럼 보이도록 인터페이스를 구성할 수 있습니다.
 
-### How do I update `Widget`s?
+### 위젯 업데이트 하기
```
```suggestion
### `위젯` 업데이트 하기
```

bro-jo commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
-This is where the concept of Stateful vs Stateless widgets
-comes in. A `StatelessWidget` is just what it sounds like&mdash;a widget with no
-state attached.
+이 지점에서 Stateful과 Stateless 위젯 컨셉이 필요합니다. `StatelessWidget`은 마치
```
```suggestion
이 지점에서 Stateful과 Stateless 위젯 개념이 필요합니다. `StatelessWidget`은 마치
```

bro-jo commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
+이해가 잘 안된다면, 이 규칙을 기억하세요: 위젯이 `build` 메소드 외부에서 변경되면(예, 런타임
+유저 상호작용의 경우) Stateful 입니다.
+상태가 한번 구성되고 더 이상 변하지 않는다면 Stateless 입니다.
+그런데, `StatefulWidget`일 지라도, 다른 변화(혹은 입력)에 반응하지 않는다면 이것을 가지고
```
```suggestion
하지만, 위젯이 stateful일지라도 위젯의 변경(혹은 다른 입력)에 의해 부모 위젯 자신이 변경되지 않는다면, 
```

bro-jo commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
+유저 상호작용의 경우) Stateful 입니다.
+상태가 한번 구성되고 더 이상 변하지 않는다면 Stateless 입니다.
+그런데, `StatefulWidget`일 지라도, 다른 변화(혹은 입력)에 반응하지 않는다면 이것을 가지고
+있는 부모 위젯은 `StatelessWidget`일 수도 있습니다.
```
```suggestion
부모 위젯은 stateless 일 수 있습니다.
```

bro-jo commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
-`StatelessWidget` is the `Text` widget. If you look at the implementation of
-the `Text` widget you'll find it subclasses `StatelessWidget`.
+아래 예제를 보고 `StatelessWidget`을 어떻게 사용하는지 살펴보세요. 일반적인
+`StatelessWidget`인 `Text` 위젯 입니다. `Text` 위젯의 구현을 살펴보면
```
```suggestion
`StatelessWidget`인 `Text` 위젯입니다. `Text` 위젯의 구현을 살펴보면
```

bro-jo commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -1,110 +1,93 @@
-the `Text` widget you'll find it subclasses `StatelessWidget`.
+아래 예제를 보고 `StatelessWidget`을 어떻게 사용하는지 살펴보세요. 일반적인
+`StatelessWidget`인 `Text` 위젯 입니다. `Text` 위젯의 구현을 살펴보면
+`StatelessWidget`의 자식클래스임을 찾을 수 있습니다.
```
```suggestion
`StatelessWidget`의 자식클래스임을 알 수 있습니다.
```

bro-jo commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -113,17 +96,15 @@ Text(
-carries no explicit state with it. It renders what is passed in its
-constructors and nothing more.
+위 코드를 살펴보면, `Text` 위젯이 아무 명시적인 상태도 전달하지 않는다는 것을 확인
+할 수 있다. 생성자에 전달된 값으로만 만들어질 뿐이다. 
```
갑자기 어미가 바뀌었네용

```suggestion
할 수 있습니다. 생성자에 전달된 값으로만 만들어질 뿐입니다. 
```

wanbok replied:
정신 놨었나봐요 ㅋ

bro-jo commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -113,17 +96,15 @@ Text(
-But, what if you want to make "I Like Flutter" change dynamically, for example
-when clicking a `FloatingActionButton`?
+하지만, "나는 Flutter가 좋아"라고 `FloatingActionButton`를 눌렀을 때 변경하려면
+어떻게 해야할까?
```
```suggestion
어떻게 해야할까요?
```

bro-jo commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -113,17 +96,15 @@ Text(
 
-But, what if you want to make "I Like Flutter" change dynamically, for example
-when clicking a `FloatingActionButton`?
+하지만, "나는 Flutter가 좋아"라고 `FloatingActionButton`를 눌렀을 때 변경하려면
```
```suggestion
하지만, `FloatingActionButton`를 눌렀을 때 "I Like Flutter"가 동적으로 변경되게 하고 싶으면
```

bro-jo commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -113,17 +96,15 @@ Text(
 
-To achieve this, wrap the `Text` widget in a `StatefulWidget` and
-update it when the user clicks the button.
+`Text` 위젯을 `StatefulWidget`으로 감싸고 유저가 버튼을 눌렀을 때 갱신하면 된다.
```
```suggestion
`Text` 위젯을 `StatefulWidget`으로 감싸고 유저가 버튼을 눌렀을 때 갱신하면 됩니다.
```

bro-jo commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -202,22 +181,19 @@ Widget build(BuildContext context) {
-widgets are immutable there is no direct equivalent to `addSubview()`.
-Instead, you can pass a function to the parent that returns a widget, and
-control that child's creation with a boolean flag.
+iOS 에서는 부모 뷰에서 `addSubview()`나 `removeFromSuperview()`를 호출하여 자식 뷰를
```
이 부분 번역이 자식뷰에서 removeFromSuperview를 호출한다고 해야 맞을 것 같습니다.
근데 제가 iOS 경험이 적어서 잘 모르겠네요. 확인 부탁드려요.

```suggestion
iOS 에서는 부모 뷰에서 `addSubview()`를 호출하거나 자식 뷰에서 `removeFromSuperview()`를 호출하여 자식 뷰를
```

wanbok replied:
@bro-jo 님 지적이 맞습니다.

bro-jo reviewed (changes requested):
코멘트 남겼습니다!

wanbok committed:
Apply @bro-jo's reviews

Update src/docs/get-started/flutter-for/ios-devs.md

Co-Authored-By: bro-jo <jhj12010@gmail.com>

Update src/docs/get-started/flutter-for/ios-devs.md

Co-Authored-By: bro-jo <jhj12010@gmail.com>

Update src/docs/get-started/flutter-for/ios-devs.md

Co-Authored-By: bro-jo <jhj12010@gmail.com>

Update src/docs/get-started/flutter-for/ios-devs.md

Co-Authored-By: bro-jo <jhj12010@gmail.com>

Update src/docs/get-started/flutter-for/ios-devs.md

Co-Authored-By: bro-jo <jhj12010@gmail.com>

Update src/docs/get-started/flutter-for/ios-devs.md

Co-Authored-By: bro-jo <jhj12010@gmail.com>

Update src/docs/get-started/flutter-for/ios-devs.md

Co-Authored-By: bro-jo <jhj12010@gmail.com>

Update src/docs/get-started/flutter-for/ios-devs.md

Co-Authored-By: bro-jo <jhj12010@gmail.com>

Update src/docs/get-started/flutter-for/ios-devs.md

Co-Authored-By: bro-jo <jhj12010@gmail.com>

Commit:
Let react-style be the origin

wanbok commented:
다시 한 번 리뷰 부탁드립니다. 어서 머지하고 아래 작업 이어가고 싶네용

bro-jo commented on src/docs/get-started/flutter-for/ios-devs.md:
```diff
@@ -202,22 +181,19 @@ Widget build(BuildContext context) {
-widgets are immutable there is no direct equivalent to `addSubview()`.
-Instead, you can pass a function to the parent that returns a widget, and
-control that child's creation with a boolean flag.
+iOS 에서는 부모 뷰에서 `addSubview()`를 호출하거나 뷰 뷰에서 `removeFromSuperview()`를
```
```suggestion
iOS 에서는 부모 뷰에서 `addSubview()`를 호출하거나 자식 뷰에서 `removeFromSuperview()`를
```

wanbok replied:
핫;

bro-jo reviewed (approved):
1곳만 수정해주시면 될 것 같습니다 :)

techhtml reviewed (approved):
@bro-jo 님이 수정 요청한 부분만 수정해주시면 될 거 같습니다 :)
LGTM.

wanbok committed:
Follow up @bro-jo's review.

Co-Authored-By: bro-jo <jhj12010@gmail.com>
